app/Support/Web3NftHandler.php

```diff
@@ -168,9 +168,14 @@ public function store(Collection $nfts, bool $dispatchJobs = false): void
         });
 
         if (Feature::active(Features::Collections->value)) {
-            CollectionModel::where('is_fetching_activity', false)->whereIn('id', $ids)->chunkById(100, function ($collections) {
-                $collections->each(fn ($collection) => FetchCollectionActivity::dispatch($collection)->onQueue(Queues::NFTS));
-            });
+            // Index activity only for newly created collections...
+            CollectionModel::query()
```

Comment:
question, does this ignore spam collections / large supplies / etc as well?

Reply:
the job itself will ignore [spam collections, large supply, etc](https://github.com/ArdentHQ/dashbrd/blob/rc-0.8.0/app/Jobs/FetchCollectionActivity.php#L54). we have scopes `withoutSpamContracts` and `withAcceptableSupply`, tho i didn't want to complicate as job will take care of that before any API calls run

 i can add these scopes though 🤷

Reply:
yeah that's fine, thanks